src/Agent.Sdk/Util/RepositoryUtil.cs

```diff
@@ -13,6 +13,7 @@ namespace Microsoft.VisualStudio.Services.Agent.Util
     public static class RepositoryUtil
     {
         public static readonly string IsPrimaryRepository = "system.isprimaryrepository";
+        public static readonly string IsTriggeringRepository = "system.istriggeringrepository";
```

Comment:
Does the server currently send this down? Or will it need to be added?

Reply:
No, this flag is completely new. For now it should default to 'self' or the single repo. We can change that behavior in the future in a couple ways.
1. Figure out the triggering repo in the agent based on some information we already have and set the flag as early as possible.
2. Let the server set this flag on the repository when it sends us the job message.

In the case of number 2, we may want to revisit the name of the flag. I was assuming we could probably keep the flag internal to the agent and do number 1, but number 2 may actually be better.